utils/src/treev2/quickPickWizard/GenericQuickPickStep.ts

```diff
@@ -21,6 +22,8 @@ export interface SkipIfOneQuickPickOptions extends GenericQuickPickOptions {
 export abstract class GenericQuickPickStep<TContext extends types.QuickPickWizardContext, TOptions extends GenericQuickPickOptions> extends AzureWizardPromptStep<TContext> {
     public readonly supportsDuplicateSteps = true;
 
+    abstract pickFilter: PickFilter<vscode.TreeItem>;

```

Comment:
```suggestion
    public readonly abstract pickFilter: PickFilter<vscode.TreeItem>;

```
Though TS might want a different order

Reply:
Made it `protected readonly`